serverlessworkflow/modules/ROOT/pages/data-index/data-index-service.adoc

```diff
@@ -0,0 +1,241 @@
+</dependency>
+----
+
+.Quarkus extension in {product_name}
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-quarkus-processes</artifactId>
+</dependency>
+----
+
+When you start your Quarkus project in development mode, an in-memory instance of the Data Index Service is automatically started in the background. This feature is enabled by link:{dev_services_url}[Quarkus Dev Services], and leverages link:{test_containers_url}[Testcontainers] to start an image of the Data Index Service.
+
+The {product_name} Process Quarkus extension sets up your Quarkus application to automatically replicate any {product_name} messaging events related to {workflow_instances} or jobs into the provisioned Data Index instance.
+
+Once the service is up and running, you can query the GraphQL interface directly using `http://localhost:8180/graphql` or using the Quarkus Dev UI console `http://localhost:8080/q/dev`.
+
+The {data_index_ref} GraphQL endpoint can query for `ProcessInstances` and `Jobs`. For more information about operations and attributes to query, see xref:data-index/data-index-core-concepts.adoc#data-index-graphql[GraphQL endpoint provided by {data_index_ref}] section.
+
+You can share the same {data_index_ref} instance across multiple {product_name} services during development. Sharing Data Index instances is enabled by default, therefore, only one Data Index instance is started. This behaviour can be adjusted to start multiple instances using the `quarkus.kogito.devservices.shared` property.
+
+The Quarkus Dev Service also allows further configuration options including:
+
+* To disable Data Index Dev Service, use the `quarkus.kogito.devservices.enabled=false` property.
+* To change the port where the Data Index Service runs, use the `quarkus.kogito.devservices.port=8180` property.
+* To adjust the provisioned image, use `quarkus.kogito.devservices.imageName=quay.io/kiegroup/kogito-data-index-ephemeral` property.
+* To disable sharing the Data Index instance across multiple Quarkus applications, use `quarkus.kogito.devservices.shared=false` property.
+
+For more information about Quarkus Dev Services, see link:{dev_services_url}[Dev Services guide].
+
+[#data-index-service-configuration-properties]
+== Data Index Service configuration properties
+The following table serves as a quick reference for commonly {data_index_ref} configuration properties supported.
+
+.Common configuration properties
+[cols="40%,35%,10%,10%,5%", options="header"]
+|===
+|Property|Description|Type|Default value|Override at runtime
+
+|`QUARKUS_DATASOURCE_JDBC_URL`
+| The datasource URL
+| string
+|
+| Yes
+
+|`QUARKUS_DATASOURCE_USERNAME`
+| The datasource username
+| string
+|
+| Yes
+
+|`QUARKUS_DATASOURCE_PASSWORD`
+| The datasource password
+| string
+|
+| Yes
+
+|`QUARKUS_DATASOURCE_DB_KIND`
+a|The kind of database to connect: `postgresql`,..
+|string
+|
+|Yes
+
+|`QUARKUS_HIBERNATE_ORM_DATABASE_GENERATION`
```

Comment:
again, I think we shouldn't mention this

Reply:
removed!